style.css

```diff
@@ -0,0 +1,92 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+html {
+    font-family: Georgia, Cambria, "Times New Roman", Times, serif;
+    background-color: #edeef0;
+}
+
+.page {
+    max-width: 1024px;
+    margin: 0 auto;
+    padding: 5px 20px 0;
+}
+
+h1 {
+    margin: 5px 0;
+    text-align: center;
+}
+
+h2 {
+    margin: 5px 0;
+}
+
+p {
+    margin: 15px;
+}
+
+li {
+    margin-left: 30px;
+    padding-left: 0;
+}
+
+footer {
```

Comment:
Let your style sheet be more structured. Footer is almost the last element of your page. Next time write it in the bottom of css file.

Reply:
@OlgaPoddubko, Yeah, stylesheets are often difficult to maintain for me. Can you suggest some rules I can follow to make it better expect this one?